src/Orleans/Configuration/ProviderConfiguration.cs

```diff
@@ -193,23 +193,39 @@ public IList<IProvider> Children
     [Serializable]
     public class ProviderCategoryConfiguration
     {
-        public string Name { get; set; }
+        public ProviderCategoryConfiguration()
```

Comment:
Should the constructors be arranged say, to be in order of visibility? Public, and then internal? In my opinion, looking at the rest of the file, it would be clearer to first have variables in order of visibility, then auto-properties, then constructors and then the rest, but one's mileage may vary and I'm not sure about what's preferred in Orleans.